apps/native/app/src/screens/wallet-pass/components/field-render.tsx

```diff
@@ -57,13 +57,13 @@ export const FieldRender = ({
               }
               return (
                 <FieldGroup key={key}>
-                  <FieldRow>
+                  <View>
```

Comment:
The license field group where rendering with way to much height and without value on Android. This fixes that